tests/Microsoft.Identity.Test.Performance/AcquireTokenForClientLargeCacheTests.cs

```diff
@@ -61,7 +61,7 @@ public async Task AcquireTokenForClient_DifferentTenants_TestAsync()
 
             await _ccaTokensDifferByTenant.AcquireTokenForClient(new[] { "scope" })
               .WithForceRefresh(false)
-              .WithAuthority($"https://login.microsoft.com/{tenant}")
+              .WithAuthority($"https://login.microsoftonline.com/{tenant}")
```

Comment:
nit: you could also define the authority on the application level to keep the same host, but if it still works, this is fine.